ui-v2/app/styles/components/healthchecked-resource/layout.scss

```diff
@@ -1,3 +1,4 @@
+%healthchecked-resource header strong,
```

Comment:
Can you explain why this was added here?

Reply:
Yup, this chunk of CSS does the truncation:

https://github.com/hashicorp/consul/blob/ba3248293644bf2985432c85be112a0edef65275/ui-v2/app/styles/components/healthchecked-resource/layout.scss#L1-L10

`%healthchecked-resource` are the 'node-cards' and the `strong` inside the `header` was the piece of text that wasn't being truncated.

I did see if I could do a `%healthchecked-resource header *` but it affected other things so I did it with a more specific selector.